fred2/sexp_tree.cpp

```diff
@@ -834,6 +834,7 @@ void sexp_tree::right_clicked(int mode)
 							case OP_CUTSCENES_SHOW_SUBTITLE:
 							case OP_ORDER:
 							case OP_TECH_ADD_INTEL:
+							case OP_TECH_REMOVE_INTEL:
```

Comment:
Look a few lines up from here where the commented-out `case OP_TECH_ADD_INTEL_XSTR` is.  Add `OP_TECH_REMOVE_INTEL_XSTR` there too.